GitUI/FormSignatureInfo.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace GitUI
+{
+    public partial class FormSignatureInfo : Form
```

Comment:
Please, inherit this form from GitExtensionsForm

Reply:
ok